project/src/main/webapp/voting_edu.html

```diff
@@ -15,18 +15,18 @@ <h2>How to Vote</h2>
     <iframe width="560" height="315" src="https://www.youtube.com/embed/
         R2yFIPRZXu4" frameborder="0" allow="accelerometer; autoplay; 
         encrypted-media; gyroscope; picture-in-picture" allowfullscreen>
-        </iframe>
+    </iframe>
```

Comment:
![image](https://user-images.githubusercontent.com/32233166/85894831-7b199000-b7c3-11ea-896f-22a1566d4748.png)
The iframe video doesn't seem to play. Could you check whether this is also occurring on your end?